src/layouts/partials/footer.html

```diff
@@ -11,7 +11,8 @@
             <div class="col-12 col-xl-3">
                 <div class="d-flex flex-row flex-wrap flex-xl-column align-items-start">
                     <a class="footer-link" href="{{ "/brand/" | relLangURL }}">{{ T "nav_brand_resources" }}</a>
-                    <a class="footer-link" href="{{ "/press/" | relLangURL }}">{{ T "nav_in_the_press" }}</a>
+                    <a class="footer-link" href="{{ "/news/"  | relLangURL }}">{{ T "nav_in_the_press" }}</a>
```

Comment:
I think it will translate to "Decred in the Press" (in English). `/news` will be a mix of own news, own press releases and 3rd party media coverage so maybe the footer link needs a change to reflect that. How about "News" or "News and Press"?

Reply:
The most flexible solution would be to have two footer links "News" and "Press" where "Press" sends to `/news/#press` and that anchor flips the toggle to only show Press. But it may need some JS so maybe we don't need that for now.

Reply:
Replaced the footer `"Decred in the Press"` link with just `"News"`